articles/azure-maps/migrate-from-bing-maps-web-services.md

```diff
@@ -528,39 +526,39 @@ The following table cross-references the Bing Maps API parameters with the compa
+* **Local search:** Searches for points of interest that are nearby (radial search), by name, or by entity type (category). The Azure Maps [POI search] and [POI category search] APIs are most like this API.
+* **Location recognition**: Searches for points of interests that are within a certain distance of a location. The Azure Maps [nearby search] API is most like this API.
+* **Local insights:** Searches for points of interests that are within a specified maximum driving time or distance from a specific coordinate. This is achievable with Azure Maps by first calculating an isochrone and then passing it into the [Search within geometry] API.
 
 Azure Maps provides several search APIs for points of interest:
 
-* [POI search](/rest/api/maps/search/getsearchpoi): Search for points of interests by name. For example; `"starbucks"`.
-* [POI category search](/rest/api/maps/search/getsearchpoicategory): Search for points of interests by category. For example; "restaurant".
-* [Nearby search](/rest/api/maps/search/getsearchnearby): Searches for points of interests that are within a certain distance of a location.
-* [Fuzzy search](/rest/api/maps/search/getsearchfuzzy): This API combines address geocoding with point of interest search. This API takes in a free-form string that can be an address, place, landmark, point of interest, or point of interest category and process the request immediately. This API is recommended for applications where users can search for addresses or points of interest from the same textbox.
-* [Search within geometry](/rest/api/maps/search/postsearchinsidegeometry): Search for points of interests that are within a specified geometry (polygon).
-* [Search along route](/rest/api/maps/search/postsearchalongroute): Search for points of interests that are along a specified route path.
-* [Fuzzy batch search](/rest/api/maps/search/postsearchfuzzybatchpreview): Create a request containing up to 10,000 addresses, places, landmarks, or point of interests and have them processed over a period of time. All the data will be processed in parallel on the server and when completed the full result set can be downloaded.
+* [POI search]: Search for points of interests by name. For example, `"starbucks"`.
+* [POI category search]: Search for points of interests by category. For example, "restaurant".
+* [Search within geometry]: Searches for points of interests that are within a certain distance of a location.
+* [fuzzy search]: This API combines address geocoding with point of interest search. This API takes in a free-form string that can be an address, place, landmark, point of interest, or point of interest category and process the request immediately. This API is recommended for applications where users can search for addresses or points of interest from the same textbox.
```

Comment:
Here, and in a few other instances, "fuzzy" is lowercased. But the lowercase is inconsistent with other instances of the word in the file. Is this change (here and in other instances of "fuzzy search") intentional or accidental?